server/routes/requests.js

```diff
@@ -11,6 +12,22 @@ router.get('/', async (req, res, next) => {
     })
 })
 
+router.patch('/applyRequesterTimezone', async (req, res, next) => {
+    try {
+        const agnosticAvailabilities = req.body.availabilities;
+        const timezone = req.body.offset
+        const availabilities = await timezoneHelpers.applyTimezone(agnosticAvailabilities, timezone)
+        res.status(200).send({
+            availabilities: availabilities
+        })
+    } catch (e) {
+        console.log(e)
```

Comment:
nit: May be we can include information about which route failed in `console.error`